src/client/tile.h

```diff
@@ -244,6 +244,8 @@ struct TileLayer
 	{
 		switch (material_type) {
 		case TILE_MATERIAL_BASIC:
+			material.MaterialType = video::EMT_SOLID;
+			break;
```

Comment:
I am not sure how I can combine them without replacing the entire switch-case block with an if-else-if cascade. I personally prefer the switch as it is more readable and extendible.